pallets/automation-time/src/lib.rs

```diff
@@ -482,6 +567,8 @@ pub mod pallet {
 					Some(task) => {
 						let action_weight = match task.action {
 							Action::Notify { message } => Self::run_notify_task(message),
+							Action::Transfer { sender, recipient, amount } =>
+								Self::run_transfer_task(sender, recipient, amount, task_id.clone()),
```

Comment:
do we need to clone task_id here?

Reply:
yah we do i think. i think when you do .iter(), it references the individual task_ids from the vector as reference, but i’m passing in an actual task_id for the run_transfer_task function. unless we want to always pass in reference for run_transfer_task function?